apps/frontend/src/reportWebVitals.ts

```diff
@@ -0,0 +1,17 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```

Comment:
Are `WebVitals` a mandatory configuration for `redux` ?

Reply:
Frère je vais te goumer inh